Makefile

```diff
@@ -160,7 +160,7 @@ dist: writeversion staticdeps staticdeps-cext buildconfig assets translation-dja
 	ls -l dist
 
 pex: writeversion
-	ls dist/*.whl | while read whlfile; do pex $$whlfile --disable-cache -o dist/kolibri-`cat kolibri/VERSION | sed -s 's/+/_/g'`.pex -m kolibri --python-shebang=/usr/bin/python; done
+	ls dist/*.whl | while read whlfile; do pex $$whlfile --disable-cache -o dist/kolibri-`cat kolibri/VERSION | sed 's/+/_/g'`.pex -m kolibri --python-shebang=/usr/bin/python; done
```

Comment:
@benjaoming do you have any idea what was the use case for  the `-s` parameter?

Anyway, this looks good to me! Tested on a Mac machine